src/components/Data/Documents/List.vue

```diff
@@ -40,7 +60,8 @@
               Here you'll see the documents in <strong>{{collection}}</strong> <br/>
               <em>Currently there is no document in this collection.</em>
             </p>
-            <button v-link="{name: 'DataCreateDocument', params: {index: index, collection: collection}}"
+            <button v-if="canCreateDocument(index, collection)"
```

Comment:
Same as above: 
We have not to hide element, but disable it with a help text.

See exemple here: [src/components/Data/Indexes/List.vue#L19](https://github.com/kuzzleio/kuzzle-backoffice/blob/master/src/components/Data/Indexes/List.vue#L19)